presto-docs/src/main/sphinx/release/release-0.284.rst

```diff
@@ -0,0 +1,67 @@
+=============
+Release 0.284
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix correctness bug in the optimizer when ``randomize_outer_join_null_key`` is enabled.
+* Fix bug in array ``CONTAINS`` expression rewrite rule.
+* Fix bugs parsing lambda expressions.  The bugs may cause compile exception with message "Can not compile special form: WHEN" and lead to runtime exception of array out of bound access.
+* Improve cache efficiency of history-based optimizer.
```

Comment:
this doesn't seem like a user facing change. If it is important can we add more details on what was the % improvement.

Reply:
sounds good. Let's keep it if there is some end benefit to the user.